src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectLaunchTargetsProvider.cs

```diff
@@ -267,6 +267,8 @@ private async Task<bool> IsIntegratedConsoleEnabledAsync()
                 }
             }
 
+            var commandLineArgs = resolvedProfile.CommandLineArgs != null ? StripCommandLineArgs(resolvedProfile.CommandLineArgs) : null;
```

Comment:
This can be simplified a little using null propagation `?.`:

```suggestion
            var commandLineArgs = resolvedProfile.CommandLineArgs?.Replace(Environment.NewLine, " ");
```

The `StripCommandLineArgs` method could then be removed.